ext/TrixiNLsolveExt.jl

```diff
@@ -95,11 +95,11 @@ function Trixi.solve_a_butcher_coeffs_unknown!(a_unknown, num_stages, monomial_c
         # Due to the nature of the nonlinear solver, different initial guesses can lead to 
         # small numerical differences in the solution.
 
-        x0 = 0.1 .* rand(rng, num_stages - 2)
+        x0 = 0.1f0 .* rand(rng, num_stages - 2)
 
         sol = nlsolve(objective_function, x0, method = :trust_region,
-                      ftol = 4e-16, # Enforce objective up to machine precision
-                      iterations = 10^4, xtol = 1e-13)
+                      ftol = 4.0f-16, # Enforce objective up to machine precision
+                      iterations = 10^4, xtol = 1.0f-13)
```

Comment:
I don't know whether `nlsolve` supports `Float32`. Did you check the results?

Reply:
Yes, and so far as I see it is the same as before.

Reply:
Hm, this looks somewhat dangerous to me - can we be sure that with this we still meet the equality constraints up to double machine precision?

Reply:
Upon reading, I think it's best to change this back to the way it initially was.